symphony-bdk-core/src/test/java/com/symphony/bdk/core/util/PresentationMLParserTest.java

```diff
@@ -0,0 +1,52 @@
+package com.symphony.bdk.core.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.symphony.bdk.core.service.message.exception.PresentationMLParserException;
+import com.symphony.bdk.core.service.message.util.PresentationMLParser;
+
+import org.junit.jupiter.api.Test;
+
+public class PresentationMLParserTest {
```

Comment:
don't we miss tests with invalid content? like empty and not even presentationml?

Reply:
it's [here](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/356/files/81cce753974f3658acb3250838badc30ec49c59c#diff-2e3993942940086fd1073a4b53029bbed8cf0fb7d78a14dc4242e452e0e80786R39)